src/ColinHDev/Elevator/Elevator.php

```diff
@@ -50,6 +52,13 @@ public function onEnable()
         $this->config = new Config($this->getDataFolder() . "config.yml", Config::YAML);
         $this->messages = new Config($this->getDataFolder() . "messages.yml", Config::YAML);
 
+        //initPermissions
+        $bypassPerm = PermissionManager::getInstance()->getPermission("elevator.*");
+        $bypassPerm->addChild("elevator.admin.use", true);
+        $bypassPerm->addChild("elevator.admin.interact", true);
+        $bypassPerm->addChild("elevator.admin.create", true);
+        $bypassPerm->addChild("elevator.admin.remove", true);
+
```

Comment:
I don't think, we should do that

Reply:
This is the way to add child-permissions

Reply:
This is unnecessary, if someone wants to set all permissions, he can do that by using those former child permissions.